web/templates/base.html

```diff
@@ -34,9 +34,36 @@
 						<li class="{% current 'pages-tasks' %}"><a href="{% url 'pages-tasks' %}">Tasks</a></li>
 						<li class="{% current 'pages-members' %}"><a href="{% url 'pages-members' %}">Members</a></li>
 						<li class="{% current 'pages-events' %}"><a href="{% url 'pages-events' %}">Events</a></li>
-						<li class="{% current 'pages-login' %}"><a href="{% url 'pages-login' %}">Login</a></li>
 						<li class="{% current 'pages-links' %}"><a href="{% url 'pages-links' %}">Help</a></li>
 					</ul>
+					<ul class="nav navbar-nav navbar-right">
+                    <li class="dropdown" id="menuLogin">
+                        {% if user.is_authenticated %}
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                                Hi, {{ user.first_name }}
+                                <b class="caret"></b>
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li><a href="#">Edit profile</a></li>
+                                <li><a href="#">Another action</a></li>
```

Comment:
How about a link to your profile? :) Also, please remove the "Something else" stuff ...